### PR TITLE
Add a README

### LINKED ISSUE #312: Add a README for top-level documentation, and as a place to link to more detailed documentation
Let us add a README.md to this repo, which will have some top-level documentation and (eventually) a pointer to more detailed documentation.

### PR DESCRIPTION
**Pull request by edwardhartnett**
Fixes #312 

Comments and additions welcome.

I'm sure my description of fv3atm in the first paragraph can be improved - what should it say?

Are there any conference papers or posters we can provide references to?

### CODE REVIEW

**Review by ligiabernardet — COMMENTED** [review]
It is great to add documentation but I am confused by the statement "This is the [NOAA Environmental Modeling Center] version of the FV3 atmospheric model dynamical core" because this repository sits above the FV3 dycore and the physics. I suggest generalizing this statement.

**Comment by edwardhartnett** [comment]
@ligiabernardet yes please. What should it say instead?

**Comment by ligiabernardet** [comment]
@edwardhartnett It will take someone with more knowledge than me about FV3ATM to write a complete statement. But here is a starting point:
This is the authoritative overarching repository for an overall driver and key subcomponents of the atmospheric component of the  [Unified Forecast System (UFS)](https://ufscommunity.org/) Weather Model. The subcomponents include the Finite-Volume Cubed-Sphere (FV3) dynamical core, originally from the [Geophysical Fluid Dynamics Laboratory](https://www.gfdl.noaa.gov/), the [Common Community Physics Package (CCPP)](https://dtcenter.org/ccpp), the [UFS stochastic physics] (https://stochastic-physics.readthedocs.io/en/latest/), ??? (missing descriptions of the io and cpl directories (if needed).

**Comment by climbfuji** [comment]
> @edwardhartnett It will take someone with more knowledge than me about FV3ATM to write a complete statement. But here is a starting point:
> This is the authoritative overarching repository for an overall driver and key subcomponents of the atmospheric component of the [Unified Forecast System (UFS)](https://ufscommunity.org/) Weather Model. The subcomponents include the Finite-Volume Cubed-Sphere (FV3) dynamical core, originally from the [Geophysical Fluid Dynamics Laboratory](https://www.gfdl.noaa.gov/), the [Common Community Physics Package (CCPP)](https://dtcenter.org/ccpp), the [UFS stochastic physics] (https://stochastic-physics.readthedocs.io/en/latest/), ??? (missing descriptions of the io and cpl directories (if needed).

stochastic physics are not part of fv3atm, they are part of the ufs-weather-model. fv3atm only contains wrapper code to call stochastic physics. The only submodules are ccpp-frramework, ccpp-physics, GFDL_atmos_cubed_sphere.

**Comment by edwardhartnett** [comment]
Thanks, I have updated based on your comments. Take another look at see what you think.

What shall we say about the io and the cpl components?

**Comment by climbfuji** [comment]
> Thanks, I have updated based on your comments. Take another look at see what you think.
> 
> What shall we say about the io and the cpl components?

They are part of fv3atm, i.e. part of the infrastructure that connects all the bits and pieces ... not sure how you want to word this.

**Comment by edwardhartnett** [comment]
Are there any papers, posters or presentations about this repo?

**Comment by ligiabernardet** [comment]
I am not aware of papers, posters or presentations about this repo.

Thank you for the changes.
Even though the ccpp-physics and ccpp-framework reside in the NCAR workspace of GitHub, it is not correct to say they are "from NCAR". The ccpp-framework and the concept for the ccpp-physics are from the Developmental Testbed Center, and many in the community have contributed to those codes (especially the physics).
Instead of 
The Common Community Physics Package (CCPP) from the [National Center
  for Atmospheric Research (NCAR)](https://ncar.ucar.edu/), including:
I suggest
The Common Community Physics Package (CCPP) supported by the [Developmental Testbed Center (DTC)](https://dtcenter.org/ccpp), including:

**Comment by edwardhartnett** [comment]
OK, I believe this is ready to merge. I need someone to approve the code review...

File: README.md
Comment (DusanJovic-NOAA) [inline README.md]: Why is Jasper required?

File: README.md
Comment (DusanJovic-NOAA) [inline README.md:29]: w3emc is required as well.

File: README.md
Comment (edwardhartnett) [reply README.md]: Carryover from where I copied this README. I've removed it.

File: README.md
Comment (junwang-noaa) [inline README.md:28]: I am not sure if this means the packages required by fv3atm or by fv3atm and its subcomponents

File: README.md
Comment (junwang-noaa) [inline README.md:43]: We also need post library

File: README.md
Comment (edwardhartnett) [reply README.md:28]: We will list only the packages directly required by fv3atm in the fv3atm README. All sub-components should do the same in their own README files.

File: README.md
Comment (edwardhartnett) [reply README.md:43]: I have added it.

**Comment by edwardhartnett** [comment]
If there are no more comments, @junwang-noaa can you approve this PR?

**Review by junwang-noaa — APPROVED** [review]
The readme looks good to me. It can be commit it after current PR.

**Comment by junwang-noaa** [comment]
@edwardhartnett Ed, it looks to me you need to sync your branch with  the top of develop branch? We can't merge the PR until your branch is updated.

**Review by DusanJovic-NOAA — APPROVED** [review]

**Review by climbfuji — APPROVED** [review]

**Comment by edwardhartnett** [comment]
I have merged the latest develop branch of fv3atm with branch.

@junwang-noaa sorry it took so long, but I didn't notice. ;-)

There is a setting that can be turned on to allow code maintainers to do these merges to branches from forks - we do this with the NCEPLIBS repos. Then this kind of update can be done without involving others, and so can happen a lot more quickly.

In a rapidly changing repo, this makes a big difference and saves this kind of waiting...

**Comment by junwang-noaa** [comment]
Ed, would you mind us combine your PR with Dusan's PR#330 (planned for tomorrow)? We have several FV3 PRs in the queue. Thanks

**Comment by edwardhartnett** [comment]
@junwang-noaa merge this in whatever way is convenient for you. Let me know if you need me to do anything further.

**Pull request merged**
